### DISCUSSION TITLE
新版本的用微云的cookie还是token

### DISCUSSION THREAD

**Discussion by GH6324** [AlistGo/alist#4966] [Q&A] [answered]
我抓到了图中的一些参数，具体用哪个参数来挂载呢
![image](https://github.com/alist-org/alist/assets/47879885/4ac333b2-dd63-4f1c-a849-ddc1d755ad69)

**Comment by anwen-anyi** [comment, marked as answer]
登录微云随意找个请求就能看到
![757be83f35c9b33d30ceb9de0505a12e](https://github.com/alist-org/alist/assets/56105412/2243a27f-60b2-4fed-8560-47ef8214724d)
